customChatWidget/README.md

```diff
@@ -183,6 +183,40 @@ Below steps explain how Custom Chat Widget works on a web page
 
 - For testing, run `npm run dev-build` to build a dev package using Babel and `webpack.dev.js` and saves the built minified file into the `public` folder with name `ACChat.js`. The dev version will have console logs.
 
+### Enabling rich messaging
+
+Amazon Connect Chat now allows your agents and customers to use rich text formatting when composing a message, enabling them to quickly add emphasis and structure to messages, improving comprehension. The available formatting options include bold, italics, hyperlinks, bulleted lists, and numbered lists. [Documentation](https://docs.aws.amazon.com/connect/latest/adminguide/enable-text-formatting-chat.html)
+
+1. To enable rich messaging, include the new param when invoking `initiateChat`:
+
+```js
+  connect.ChatInterface.initiateChat({
+    contactFlowId: "${contactFlowId}",
+    instanceId: "${instanceId}",
+    // ...
+    supportedMessagingContentTypes: "text/plain,text/markdown", // include 'text/markdown' for rich messaging support
+    featurePermissions: {
+      ATTACHMENTS: false,
+      MESSAGING_MARKDOWN: true
+    }
+  },successHandler, failureHandler)
+```
+
+2. If updating an exisiting CFN stack, the startChatContact lambda function needs to be updated.
```

Comment:
Since this README is outside the Cloudformation folder, can we make this more clear that updating(redeploying) existing CFN stack is required to update lambda and SDK? And customer can check their lambda code to tell if they need to update the stack or not.

Reply:
Yes that will be more clear. Added link to lambda function and file path to snippet.